### PR TITLE
Update a test to work with recent markdown

### PR DESCRIPTION
**Pull request by yanok**
`custom_syntax_test` needs to be updated, since
https://github.com/dart-lang/markdown/commit/d72ae07c8290b3780044170eda28eda5a9fb342e change in the `markdown` package won't allow changing `children` of the `Element` created with `Element.withTag()`.

### CODE REVIEW

File: packages/flutter_markdown/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## NEXT
```
Comment (domesticmouse) [inline packages/flutter_markdown/CHANGELOG.md:1]: Can you please mark this as 0.6.13 and update the `pubspec.yaml` as well?

**Review by domesticmouse — CHANGES_REQUESTED** [review]

File: packages/flutter_markdown/CHANGELOG.md (hunk 1 above)
Comment (yanok) [reply packages/flutter_markdown/CHANGELOG.md]: Done.

**Review by domesticmouse — APPROVED** [review]

**Comment by domesticmouse** [comment]
PTAL @stuartmorgan

File: packages/flutter_markdown/CHANGELOG.md (hunk 1 above)
Comment (stuartmorgan) [reply packages/flutter_markdown/CHANGELOG.md]: > Can you please mark this as 0.6.13 and update the `pubspec.yaml` as well?

Why? This is a test-only change, so it doesn't require a version bump.

Comment (stuartmorgan) [reply packages/flutter_markdown/CHANGELOG.md]: Yes, the version bump should be reverted. Even the changelog is optional here; the repo policy changed somewhat recently to not require changelog entries for package-developer-only changes (such as changes that only affect tests).

**Comment by yanok** [comment]
Abandoning this PR, the issue is already fixed by 3c838819a2e2d4178041c55c04b143c24aba9618

**Pull request closed without merging**